### DISCUSSION TITLE
No red X when modifying `.github/workflows`

### DISCUSSION THREAD

**Discussion by jackkoenig** [Mergifyio/mergify#1652] [General]
Whenever I have a PR that modifies `.github/workflows` that would otherwise be merged by Mergify, I get a red X and the message "GitHub App like Mergify are not allowed to merge pull request where `.github/workflows` is changed." The reasoning and resolution make sense and I'm happy to merge on my own, but would it be possible to not get a Red X? Maybe a yellow exclamation point instead?

Just would like to add that I love you work and I appreciate everything y'all do!

**Comment by jd** [comment]
Mergify returns an `action_required` (as one can see in [source code](https://github.com/Mergifyio/mergify-engine/blob/master/mergify_engine/actions/merge_base.py#L540)) when that happens.

That ought to be a warning symbol or the like as shown in the [docs](https://docs.github.com/en/free-pro-team@latest/rest/guides/getting-started-with-the-checks-api#about-check-suites):

![check suites](https://docs.github.com/assets/images/check_suites.png)

We don't control what is shown in GitHub UI though. I don't really get why they show a red cross there unfortunately.

**Reply by jackkoenig** [reply]
I see, well it looks like you're doing the right thing so I'll take my question / feature request to GitHub! Thanks!

I noticed that the `Action Required` updates to a `Success` once I've merged manually, so at least it becomes a green checkmark after merging, so that's good. Thanks again!
